test/functional-tests/plan.md

```diff
@@ -0,0 +1,404 @@
+# Parameter Framework (PF) test plan
+
+## Notes
+- The unit test code MAY be able to generate supported test scenario
+  in the same format to easly check for scenario scenario coverage.
+- All scenarios implicitly start by _"GIVEN a parameter framework"_.
+- If a scenario does not specify a config files,
+  the following minimal files SHOULD be used:
+
+Top level file (replace `{structurePath}` by it's value):
+~~~ xml
+<?xml version='1.0' encoding='UTF-8'?>
+<ParameterFrameworkConfiguration SystemClassName='test'
+                                 TuningAllowed='false' ServerPort='1'>
+    <SubsystemPlugins/>
+    <StructureDescriptionFileLocation Path='{structurePath}'/>
+</ParameterFrameworkConfiguration>
+~~~
+
+Structure file:
+~~~ xml
+<?xml version='1.0' encoding='UTF-8'?>
+<SystemClass Name='test'>
+    <Subsystem Name='test' Type='Virtual' Endianness='Little'>
+        <ComponentLibrary/>
+        <InstanceDefinition>
+            <BooleanParameter Name="placeholder"/>
```

Comment:
Also, it is not what is currently implemented: the default structure file does not contain any parameter.